src/main.ts

```diff
@@ -229,6 +241,40 @@ app.on('ready', () => {
 	mainWindow.on('ready-to-show', () => {
 		if (userPrefs.fullscreen === 'maximized' && !mainWindow.isMaximized()) mainWindow.maximize();
 		if (!mainWindow.isVisible()) mainWindow.show();
+		const filter: WebRequestFilter = { urls: [] };
```

Comment:
probably move this somewhere outside - the mainWindow.on(ready-to-show) callback is kinda full imo, i'd extract this outside, atleast the filter definition